examples/seismic/tti/tti_example.py

```diff
@@ -71,6 +64,29 @@ def test_tti_stability(shape, kernel):
     assert np.isfinite(norm(rec))
 
 
+@pytest.mark.parametrize('shape', [(51, 51), (16, 16, 16)])
+def test_model_updt(shape):
```

Comment:
Some nitpicking:
- Can this be moved in `model.py` 
- Can we test all cases, i.e 1) replacing the value of existing param by an array, 2) replacing existing param with a new Function 3) new param array, 4) new param Function